jekyll/_cci2/container-runner.adoc

```diff
@@ -304,6 +317,19 @@ Container agent will drain and restart cleanly when sent a termination signal. A
 
 At this time, if container agent crashes, there is no expectation that in-process or queued tasks are handled gracefully. As the open preview progresses, additional crash handling will be added and documented.
 
+[#constraint-validation]
+== Constraint Validation
+
+Container agent allows you to configure tasks pods with the full range of Kubernetes settings. This potentially means pods can be configured in a way which cannot be scheduled due to their constraints. To help with this container agent has a constraint checker which periodically validates each resource class configuration against the current state of the cluster, to ensure pods can be scheduled.
```

Comment:
~ I'd add in this is so that tasks (maybe jobs since a task isn't a concept we talk about publicly much) aren't claimed when they can't be ran, and then subsequently failed because of that.